ASF-STM.user.js

```diff
@@ -947,10 +947,11 @@
             if (userindex == -1) {
                 profileLink = myProfileLink;
                 updateMessage("Getting our data for badge " + (index + 1) + " of " + botBadges.length);
+                updateProgress(index, botBadges.length);
             } else {
                 profileLink = "profiles/" + bots.Result[userindex].SteamID;
                 updateMessage("Fetching bot " + (userindex + 1).toString() + " of " + bots.Result.length.toString() + " (badge " + (index + 1) + " of " + botBadges.length + ")");
-                updateProgress(userindex);
+                updateProgress(userindex, botBadges.length)
```

Comment:
It should be `bots.Result.length`, not `botBadges.length`. Please, at least test your script before submitting PR, it's easy to notice.